docs/learn/universal-profile/metadata/edit-profile.md

```diff
@@ -230,7 +149,7 @@ To set up the erc725.js library, we will need the following:
 
 - The address of our Universal Profile contract: this is the address of our profile mentioned in the URL on the [profile explorer](https://universalprofile.cloud/).
 - An ERC725Y JSON Schema: a set of ERC725Y key-value pairs ([LSP2 - ERC725Y JSON Schema](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md))
-- A RPC provider: `https://rpc.testnet.lukso.network`
+- A RPC provider: `https://4201.rpc.thirdweb.com`
 
 Once our erc725.js is initialized, we can encode the `LSP3Profile` data to generate a key and a value.
```

Comment:
Suggest rewording to:

Once our erc725.js is initialized, we can encode the `LSP3Profile` data key using the [`encodeData()`](../../../tools/erc725js/methods.md#encodeData) function. We need to call it with an object parameter containing the `keyName` and the `value` as follow:

Reply:
Also this sentence can be deleted

<img width="805" alt="image" src="https://github.com/user-attachments/assets/c0be72fd-9d60-458c-a62a-c804cd149b80">